content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -37,208 +34,375 @@ localization:
 
-.. image:: colombia/carvajal-credential-config.png
-   :alt: Configure credentials for Carvajal web service in Odoo.
+To configure your company information, go to the :menuselection:`Contacts` app and search for your
+company or activate the :ref:`developer mode <developer-mode>` and go to :menuselection:`Company
+--> Contact`. Then, edit the **contact form** to configure the following information:
 
-Check the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment. This
-allows users to test the complete workflow and integration with the :abbr:`CEN (Centro Electrónico
-de Negocios)` Financiero portal, which is accessible here:
+#. General data
 
-- `CTS (Carvajal T&S) <https://cenflab.cen.biz/site/>`_.
-- `CSC (Carvajal Servicios de Comunicación) <https://web-stage.facturacarvajal.com/>`_.
+   - :guilabel:`Name`.
+   - :guilabel:`Address`: Including :guilabel:`City`, :guilabel:`Department` and
+     :guilabel:`Zip Code`.
+   - :guilabel:`Tax ID`: When it is a `NIT`, it must have the *verification digit* at the end of the
+     ID followed by a hyphen (`-`).
 
-:abbr:`CSC (Carvajal Servicios de Comunicación)` is the default for new databases.
+   .. image:: colombia/company-configuration.png
+      :alt: Configure your company contact form in Odoo.
 
-Once Odoo and Carvajal are fully configured and ready for production, the testing environment can be
-disabled by unchecking the :guilabel:`Test mode` checkbox.
+#. :guilabel:`Fiscal Information`
 
-Configure report data
----------------------
+   In the :guilabel:`Sales & Purchase` tab, configure the following information:
+
+   - :guilabel:`Fiscal Position`: Determines the taxes/accounts used for this contract.
+   - :guilabel:`Tipo de Representación`.
+   - :guilabel:`Tipo Establecimiento`.
+   - :guilabel:`Obligaciones y Responsabilidades`.
+   - :guilabel:`Usuario Aduanero`.
+   - :guilabel:`Gran Contribuyente`.
+   - :guilabel:`Fiscal Regimen`.
+   - :guilabel:`Commercial Name`.
+
+   .. image:: colombia/fiscal-information.png
+      :alt: Configure the fiscal information of your company in Odoo.
+
+Carjaval credentials configuration
+----------------------------------
+
+Once the modules installed, the user credentials must be configured in order to connect with
+Carvajal Web Service. Navigate to :menuselection:`Accounting --> Configuration --> Settings` and
+scroll to the :guilabel:`Colombian Electronic Invoicing` section. Then, fill in the required
+configuration information provided by Carvajal:
+
+- :guilabel:`Username` and :guilabel:`Password`: Correspond to the username and password provided
+  by Carvajal to the company.
+- :guilabel:`Company Registry`: Company's NIT number *without* the verification code.
+- :guilabel:`Account ID`: Company ID followed by `_01`.
+- :guilabel:`Colombia Template Code`: Select one of the two available templates (:guilabel:`CGEN03`
+  or :guilabel:`CGNE04`) to be used in the PDF format of the electronic invoice.
```

Comment:
```suggestion
- :guilabel:`Colombia Template Code`: Select one of the two available templates (:guilabel:`CGEN03`
  or :guilabel:`CGNE04`) to be used in the PDF format of the electronic invoice.
```
I do not know if the template is specified by the Carvajal or not, but if the user is free to choose, it could be nice to add the difference (i.e. the reasons to choose one or the other)

Reply:
> EWA comment:  The template is defined by Carvajal, and the client determines which one to use with Carvajal. We just sent the data to Carvajal.